src/org/linphone/HistoryDetailFragment.java

```diff
@@ -155,11 +192,11 @@ public void onResume() {
 	public void onClick(View v) {
 		int id = v.getId();
 		
-		if (id == R.id.dialBack) {
+		if (id == R.id.dial) {
 			LinphoneActivity.instance().setAddresGoToDialerAndCall(sipUri, displayName, pictureUri == null ? null : Uri.parse(pictureUri));
```

Comment:
If picture uri is null, you might want to pass a placeholder picture or simply not add the picture at all. Might cause a null pointer exception down the line.